TheFourFifths/consus
# Checking for overdue items. [TFF-78]

### Summary

Changed student store to hold items with students instead of just addresses. Added timestamps for due time when an item gets checked out. Checks that the students don't have overdue items before allowing a successful checkout.

Note: This PR has a sister PR on the consus-client repo. They should be merged together or not at all.


### Checklist


Finally, is it ready to be reviewed and merged: yes :white_check_mark: 

### How to Review

1. Check that new functionality doesn't interfere with anything. 
2. Ensure added functionality makes sense. 

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-78](https://msoese.atlassian.net/browse/TFF-78)
